movement/watch_faces/complication/sunrise_sunset_face.c

```diff
@@ -197,6 +197,8 @@ static void _sunrise_sunset_face_update_settings_display(movement_event_t event,
     char buf[12];
 
     switch (state->page) {
+        case 0:
+            return;
```

Comment:
Future improvement ideas: the page numbers with could be replaced with `enum` constants. The compiler can warn when `switch` statements don't cover all enumerated constants and also when they lack a `default` case.